index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <meta charset="utf-8" />
+      <title>Carrera Programmer - Portfolio</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,100;1,300;1,700&family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+      <link rel="stylesheet" href="styles.css">
+      <script src="https://kit.fontawesome.com/321e54ee43.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+      <header>
+        <nav class="navbar">
+            <p class="logo">Carrera<i  id=logo-dat class="fa-solid fa-circle-dot"></i>Programmer</p>
```

Comment:
- [ ] The logo should be on the left of the top bar and has a link (a tag), kindly wrap it on an `<a>` tag so it's clickable